main.cpp

```diff
@@ -141,6 +141,14 @@ int main(int argc, char *argv[])
 
     // loglevel is configured in main.qml. Anything lower than
     // qWarning is not shown here.
+    QString logLevelStr = qgetenv("MONERO_LOG_LEVEL");
```

Comment:
Use
```c++
int qEnvironmentVariableIntValue(const char *varName, bool *ok=nullptr) noexcept`
```

Reply:
Aah that looks good, thanks for suggestion!